build/configs/artik053/grpc/defconfig

```diff
@@ -421,7 +421,8 @@ CONFIG_NETDEV_MULTINIC=y
 # CONFIG_NET_E1000 is not set
 # CONFIG_NET_SLIP is not set
 # CONFIG_NET_VNET is not set
-# CONFIG_PIPES is not set
+CONFIG_PIPES=y
```

Comment:
Why did you change this configuration?

Reply:
I enabled the PIPES feature to avoid copilation errors.